srcpkgs/unrar/template

```diff
@@ -12,13 +12,12 @@ checksum=505c13f9e4c54c01546f2e29b2fcc2d7fabc856a060b81e5cdfe6012a9198326
 repository=nonfree
 
 do_build() {
-	vsed -e 's/^\(CXXFLAGS\)=\(.*\)/\1+=\2/' \
+	vsed -e 's/^\(CXXFLAGS\)=-march=native \(.*\)/\1+=\2/' \
```

Comment:
This is assuming  '-march=native' is always the first in CXXFLAGS
Is that a safe assumption ?

Reply:
Yes, for version 7.0.9 it is (upstream makefile has it hardcoded like that), I wouldn't expect them to re-release the same version with the flag (re)moved.

Reply:
OK, as-is it' no problem, but if upstream changes that in the future, it may be a (slight) trap for the one updating the void packaging. But that does not look like a big problem, just if it does not match the regex, the flag will be used again... Feel free to ignore this comment as well.